groups/groups.yaml

```diff
@@ -209,3 +209,8 @@ groups:
       - ihor@cncf.io
       - justinsb@google.com
       - thockin@google.com
+  - email-id: k8s-infra-staging-kube-state-metrics@kubernetes.io
```

Comment:
The max number of characters can only be 30. Ref: https://github.com/kubernetes/k8s.io/pull/289#issuecomment-508806417. :grimacing:

Reply:
(side note: we should add validation for this)

Reply:
shortened to `k8s-infra-staging-kube-metrics`, with a comment explaining why it's shortened

Reply:
shortened to k8s-infra-staging-kube-metrics, with a comment explaining why it's shortened